cmd/bb_storage/main.go

```diff
@@ -52,12 +52,16 @@ func main() {
 		var contentAddressableStorageInfo *blobstore_configuration.BlobAccessInfo
 		var contentAddressableStorage blobstore.BlobAccess
 		if configuration.ContentAddressableStorage != nil {
+			supportedCompressors := global.GetSupportedCompressors(configuration.Global)
+			casCreator := blobstore_configuration.NewCASBlobAccessCreator(
+				grpcClientFactory,
+				int(configuration.MaximumMessageSizeBytes),
+				supportedCompressors)
+
 			info, authorizedBackend, allAuthorizers, err := newScannableBlobAccess(
 				dependenciesGroup,
 				configuration.ContentAddressableStorage,
-				blobstore_configuration.NewCASBlobAccessCreator(
-					grpcClientFactory,
-					int(configuration.MaximumMessageSizeBytes)),
+				casCreator,
```

Comment:
Why can't we get rid of all of these changes, but change this line further below:
```go
cacheCapabilitiesProviders = append(cacheCapabilitiesProviders, info.BlobAccess)
```
To:
```go
cacheCapabilitiesProviders = append(
        cacheCapabilitiesProviders,
        info.BlobAccess,
        capabilities.NewStaticProvider(&remoteexecution.ServerCapabilities{
		CacheCapabilities: &remoteexecution.CacheCapabilities{
 			SupportedCompressors: configuration.SupportedCompressors,
 		},
        }),
)
```
?

Reply:
Nice, this merged provider is awesome